index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hijab Center</title>
```

Comment:
Extra Feature: You can Add Webpage icon(Favicon) on the tab by using this. You can download tons of options from "https://icons8.com/icons/set/hijab", you can download png file and write it's address in the href
```suggestion
  <link type="image/png" sizes="16x16" rel="icon" href=".../icons8-woman-with-headscarf-16.png">
```

Reply:
yup i try this site.. now i use logo which is taken from this website,  i want my logo attach to my title thats why here i use span tag in h1 tag..

Reply:
and img in it for logo.

Reply:
and here i use href ..cdn